docker-compose.yml

```diff
@@ -1,37 +1,38 @@
-# docker-compose.yml
 version: '3'
 networks:
   todoapp-net:
 services:
   pythonapp:
-    image: todoapp:2.0.0
+    image: volry/todoapp:1.0.0
```

Comment:
![image](https://github.com/user-attachments/assets/2e76e6b4-8bec-47ce-b85a-147a0e9e871a)
Need to update version of container